src/python/foglamp/configuration_manager.py

```diff
@@ -29,7 +31,8 @@
 
 _valid_type_strings = ['boolean', 'integer', 'string', 'IPv4', 'IPv6', 'X509 certificate', 'password']
```

Comment:
Got it, I was tuned to _valid_type_strings password context only. @ashwinscale  could you please fix and test that `connection_string = "dbname='foglamp'"` in a new PR for `configuration_manager.py`? I will merge that and rebase this PR with develop.

> This is where  DB model  (interim solution for storage) layer https://github.com/foglamp/FogLAMP/pull/76/files shines, BTW. We are using conn_string in sensor_value, omf_translator and configuration_manager in develop branch; and it will be in scheduler, purge and other places as needed, so is for the same tables meta info e.g. readings. So having that at single place, will allow schema update (ddl match) etc. easy to maintain/ change. Just import that and later this will be easy to plug out this *interim* layer and plug in storage_engine methods which itself has an abstraction for other services for DB.